cartographer/mapping/internal/2d/pose_graph_2d.cc

```diff
@@ -1077,5 +1084,12 @@ void PoseGraph2D::SetGlobalSlamOptimizationCallback(
   global_slam_optimization_callback_ = callback;
 }
 
+void PoseGraph2D::RegisterMetrics(metrics::FamilyFactory* family_factory) {
+  auto* latency = family_factory->NewGaugeFamily(
+      "mapping_internal_2d_pose_graph_work_queue_delay",
```

Comment:
I wonder why `internal` is added here (does not seem to provide useful information), but the redundant `2d` is left out. Maybe we should either include all information or remove noise, i.e. also `internal`. What do you think?

Reply:
I have literally no opinion on this, since this is Cartographer-internal convention. I just took the style from local_trajectory_builder_2d, which doesn't repeat "2d" either.
@cschuet What do you think?

Reply:
@gaschler You have to decide :)

Reply:
@CodeArno Sounds fine. Remove it here and then everywhere else in a later PR. (On the long run, of course, metric labels should be stable.)

Reply:
Okay, sounds good.